cmd/snap/cmd_advise.go

```diff
@@ -107,7 +108,33 @@ type jsonRPC struct {
 		Command         string   `json:"command"`
 		SearchTerms     []string `json:"search-terms"`
 		UnknownPackages []string `json:"unknown-packages"`
-		Packages        []string `json:"packages"`
+		Packages        []struct {
```

Comment:
Couldn't that be replaced with
```
Packages interface{} `json:"packages"`
```

Reply:
Yes, that would work as well. Probably the right answer to the problem (YAGNI and all that) - and we can add the full structure once we actually use it.

Reply:
Or just drop the field altogether. The JSON parser will ignore it then.